pkg/config/lima_config_applier.go

```diff
@@ -43,6 +44,23 @@ func (lca *limaConfigApplier) Apply() error {
 
 	limaCfg.CPUs = lca.cfg.CPUs
 	limaCfg.Memory = lca.cfg.Memory
+	limaCfg.Mounts = []limayaml.Mount{}
+	for _, ad := range lca.cfg.AdditionalDirectories {
+		// Copied default configs from Lima code:
+		// https://github.com/lima-vm/lima/blob/142c5fd17c31464e6d37ac2d78b24c16a5500968/pkg/limayaml/defaults.go#L448.
+		limaCfg.Mounts = append(limaCfg.Mounts, limayaml.Mount{
+			Location: *ad.Path, MountPoint: *ad.Path, Writable: pointer.Bool(true),
+			SSHFS: limayaml.SSHFS{
+				Cache: pointer.Bool(true), FollowSymlinks: pointer.Bool(false),
+				SFTPDriver: pointer.String(""),
+			},
+			NineP: limayaml.NineP{
+				SecurityModel:   pointer.String(limayaml.Default9pSecurityModel),
+				ProtocolVersion: pointer.String(limayaml.Default9pProtocolVersion),
+				Msize:           pointer.String(limayaml.Default9pMsize), Cache: pointer.String(limayaml.Default9pCacheForRW),
+			},
```

Comment:
It seems that we may only need to specify `Location` and `Writable` and omit all the other fields (i.e., effectively leaving them `null`) according to https://github.com/lima-vm/lima/blob/master/examples/default.yaml? We then don't have to check those fields in tests. We're just specifying the default values here anyway.

Reply:
Yeah my bad. I didn't notice the final mount is applied the default values after merging the override config.